src/auth/strategies/jwt.strategy.ts

```diff
@@ -1,30 +1,28 @@
 import { PassportStrategy } from '@nestjs/passport';
 import { ExtractJwt, Strategy } from 'passport-jwt';
-import { AuthService } from '../auth.service';
-import { TOKENS } from '@/config';
 import { HttpException, HttpStatus, Injectable } from '@nestjs/common';
-import { JwtPayload } from '../types/jwt-payload';
 import { paths } from '@/generated/schema';
+import { JwtPayload } from '../types/jwt-payload';
 import { PasswordOmitUsers } from '../types/passwordOmitUsers';
+import { UsersService } from '@/users/users.service';
 
 @Injectable()
 export class JwtStrategy extends PassportStrategy(Strategy) {
-	constructor(private readonly authService: AuthService) {
+	constructor(private readonly usersService: UsersService) {
 		super({
 			jwtFromRequest: ExtractJwt.fromAuthHeaderAsBearerToken(),
 			ignoreExpiration: false,
-			secretOrKey: TOKENS.ACCESS_TOKEN_SECRET,
+			secretOrKey: process.env.JWT_ACCESS_TOKEN_SECRET,
```

Comment:
ここは定数のレイヤーに戻して大丈夫そう？(命名は現状の実装ベースで例示します

```suggestion
			secretOrKey: TOKENS.JWT_ACCESS_TOKEN_SECRET,
```

Reply:
戻した！